Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2700 -- Modify admin navigation menu order and capitalization

Resolves [EREGCSC-2700](https://jiraent.cms.gov/browse/EREGCSC-2700)

**Description**

Reorder items in "Resources for Regulations" list wherever it is shown in the Admin panel while slightly tweaking the casing of the list items.

**This pull request changes:**

- Adds `admin_priority` model field to match order in AC
- adds `verbose_name` and `verbose_name_plural` `Meta` options in `internal_resources` model to update the list item label casing for internal resource list items.  This matches the pattern used for all other list items in this list.

**Steps to manually verify this change:**

1. Green check marks
2. Log in to the experimental deployment's admin panel
3. Compare the "Resources for Regulations" list items in the admin panel and ensure that they match the order and label casing of the list in the JIRA ticket
4. If you're validating with a user that has superuser status or full admin rights, you will also see "Sections" and "Subparts" at the bottom of the list
